diff --git a/src/main/java/com/pilates/booking/web/rest/errors/EmailAlreadyUsedException.java b/src/main/java/com/pilates/booking/web/rest/errors/EmailAlreadyUsedException.java
index ff84921fa81965cdfde9e1abcccd8c011c0d0e2a..f6387191383c98b78ae7c40b8fff78c34113e4ad 100644
--- a/src/main/java/com/pilates/booking/web/rest/errors/EmailAlreadyUsedException.java
+++ b/src/main/java/com/pilates/booking/web/rest/errors/EmailAlreadyUsedException.java
@@ -1,11 +1,29 @@
 package com.pilates.booking.web.rest.errors;
 
+import org.springframework.http.HttpStatus;
+import org.springframework.web.ErrorResponseException;
+import tech.jhipster.web.rest.errors.ProblemDetailWithCause;
+import tech.jhipster.web.rest.errors.ProblemDetailWithCause.ProblemDetailWithCauseBuilder;
+
 @SuppressWarnings("java:S110") // Inheritance tree of classes should not be too deep
-public class EmailAlreadyUsedException extends BadRequestAlertException {
+public class EmailAlreadyUsedException extends ErrorResponseException {
 
     private static final long serialVersionUID = 1L;
 
     public EmailAlreadyUsedException() {
-        super(ErrorConstants.EMAIL_ALREADY_USED_TYPE, "Email is already in use!", "userManagement", "emailexists");
+        super(
+            HttpStatus.CONFLICT,
+            ProblemDetailWithCauseBuilder.instance()
+                .withStatus(HttpStatus.CONFLICT.value())
+                .withType(ErrorConstants.EMAIL_ALREADY_USED_TYPE)
+                .withTitle("Email is already in use!")
+                .withProperty("message", "EMAIL_ALREADY_USED")
+                .build(),
+            null
+        );
+    }
+
+    public ProblemDetailWithCause getProblemDetailWithCause() {
+        return (ProblemDetailWithCause) this.getBody();
     }
 }
